src/tester.rs

```diff
@@ -85,6 +85,18 @@ impl<'a> LangTester<'a> {
         self
     }
 
+    /// Specify the number of simultaneous running test cases. By default, Rust
+    /// will attempt to use all available CPUs.
+    pub fn test_threads(&'a mut self, mut test_threads: usize) -> &'a mut Self {
+        if test_threads > num_cpus::get() {
```

Comment:
I don't think we want this check: if someone wants to specify a large number of test_threads, we should trust them that they have a good reason for doing so (e.g. perhaps many of them wait on the network).

Reply:
That makes sense. I've removed it here: 63a315797f36da4cb4212cbafdf3b3d52253d8f9